pkg/systemd/logsJournal.jsx

```diff
@@ -168,7 +168,7 @@ export class JournalBox extends React.Component {
             const serviceTag = entries[i].SYSLOG_IDENTIFIER;
             this.renderer.prepend(entries[i]);
             // Only update if the service is not yet known
-            if (serviceTag && !this.props.currentIdentifiers.includes(serviceTag))
+            if (serviceTag && this.props.currentIdentifiers && !this.props.currentIdentifiers.includes(serviceTag))
```

Comment:
This looks fishy. Without knowing the context, I would expect a falsy `currentIdentifiers` to be treated like the empty list. I.e.:
```
!(this.props.currentIdentifiers || []).includes(serviceTag)
```

Reply:
But it's probably better to initialize `this.props.currentIdentifiers` to [] explicitly, in the parent component.